mpqp/core/instruction/gates/native_gates.py

```diff
@@ -182,6 +183,14 @@ def braket_gate(
     ]:
         pass
 
+    @classproperty
```

Comment:
isn't it a problem for the typechecker to put these two together ?

Reply:
No, it isn't a problem